index.html

```diff
@@ -3246,6 +3246,55 @@ <h3>Authorization</h3>
 on top of this specification.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Reserved Extension Points</h3>
+
+        <p>
+This specification reserves a number of extension point <a>properties</a> that
+implementers signalled interest in, where their inclusion in this specification
+was considered to be premature, and where these extension points might be more
+formally defined in future versions of this specification. While these extension
```

Comment:
This is purely a stylistic comment and, as such, is a matter of taste...

```suggestion
This specification reserves a number of <a>properties</a> serving as possible extension points. While
some implementers signaled interest in these, their inclusion in this specification
was considered to be premature; these extension points might be more
formally defined in future versions of this specification. While these extension
```